full-service/src/db/encryption_provider.rs

```diff
@@ -0,0 +1,256 @@
+// Copyright (c) 2020 MobileCoin Inc.
```

Comment:
@eranrund - refactored the encryption_provider functionality out of the wallet_db. LMK what you think!